src/main/java/seedu/address/logic/parser/CustomiseCommandParser.java

```diff
@@ -85,23 +93,57 @@ private Attribute[] toAttributeOrder(String[] args) throws ParseException {
      * @return the Attribute array order
      */
     private Attribute[] fillMissingAttributes(Attribute[] attributeArr, boolean[] isAttributeUsed, int noOfMissing) {
-        for (int i = 1; i <= noOfMissing; i++) {
+        int indexOfLastAttribute = attributeArr.length - 1;
+        for (int i = 0; i < noOfMissing; i++) {
             if (!isAttributeUsed[0]) {
-                attributeArr[4 - i] = Attribute.ADDRESS;
+                attributeArr[indexOfLastAttribute - i] = Attribute.ADDRESS;
                 isAttributeUsed[0] = true;
             } else if (!isAttributeUsed[1]) {
-                attributeArr[4 - i] = Attribute.EMAIL;
+                attributeArr[indexOfLastAttribute - i] = Attribute.EMAIL;
                 isAttributeUsed[1] = true;
             } else if (!isAttributeUsed[2]) {
-                attributeArr[4 - i] = Attribute.PHONE;
+                attributeArr[indexOfLastAttribute - i] = Attribute.PHONE;
                 isAttributeUsed[2] = true;
             } else if (!isAttributeUsed[3]) {
-                attributeArr[4 - i] = Attribute.TAGS;
+                attributeArr[indexOfLastAttribute - i] = Attribute.TAGS;
                 isAttributeUsed[3] = true;
             } else {
                 assert false;
             }
```

Comment:
The default order of Attributes here appear to be different from the one in GuiSettings

Reply:
I have changed the default order in GuiSettings to `TAGS>PHONE>EMAIL>ADDRESS`.